scripts/components/matrix-align-control/matrix-align-control.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { __experimentalBlockAlignmentMatrixControl as BlockAlignmentMatrixControl } from '@wordpress/block-editor';
+import { __experimentalBlockAlignmentMatrixToolbar as BlockAlignmentMatrixToolbar } from '@wordpress/block-editor';
```

Comment:
This will work in all cases? Like in 5.8 they use one in 5.7 they use another, but this won't throw some undefined import error if this is missing.

I think in the docs we should just note that we need to look when they stop making this as experimental 😅

Reply:
There's only a warning in Storybook, no errors in the browser as you never try to use the non-existant component.